aptos-move/framework/aptos-stdlib/sources/cryptography/algebra_bls12381.move

```diff
@@ -1,11 +1,214 @@
 /// This module defines marker types, constants and test cases for working with BLS12-381 curves
 /// using generic API defined in `algebra.move`.
 ///
-/// Currently supported BLS12-381 structures include field `Fr`.
+/// See https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-pairing-friendly-curves-11#name-bls-curves-for-the-128-bit-
+/// for the full specification of BLS12-381 curves.
+///
+/// Currently-supported BLS12-381 structures include `Fq12`, `Fr`, `G1Affine`, `G2Affine` and `Gt`,
+/// along with their widely-used serialization formats,
+/// the pairing between `G1Affine`, `G2Affine` and `Gt`,
+/// and the hash-to-curve operations for `G1Affine` and `G2Affine` defined in https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-hash-to-curve-16.
 module aptos_std::algebra_bls12381 {
     //
-    // Marker types and their serialization schemes begin.
+    // Marker types + serialization formats begin.
     //
+    /// The finite field $F_q$ used in BLS12-381 curves.
+    /// It has a prime order $q$ equal to 0x1a0111ea397fe69a4b1ba7b6434bacd764774b84f38512bf6730d2a0f6b0f6241eabfffeb153ffffb9feffffffffaaab.
+    ///
+    /// NOTE: currently information-only and no operations are implemented for this structure.
+    struct Fq {}
+
+    /// A serialization format for `Fq` elements,
+    /// where an element is represented by a byte array `b[]` of size 48 with the least signature byte coming first.
+    ///
+    /// NOTE: currently information-only, not implemented.
+    struct FqFormatLsb {}
+
+    /// A serialization format for `Fq` elements,
+    /// where an element is represented by a byte array `b[]` of size 48 with the most significant byte coming first.
+    ///
+    /// NOTE: currently information-only, not implemented.
+    struct FqFormatMsb {}
+
+    /// The finite field $F_{q^2}$ used in BLS12-381 curves.
+    /// It is an extension field of `Fq`, constructed as $F_{q^2}=F_q[u]/(u^2+1)$.
+    ///
+    /// NOTE: currently information-only and no operations are implemented for this structure.
+    struct Fq2 {}
+
+    /// A serialization format for `Fq2` elements.
+    /// where an element in the form $(c_0+c_1\cdot u)$ is represented by a byte array `b[]` of size 96
+    /// with the following rules.
+    /// - `b[0..48]` is $c_0$ serialized using `FqFormatLsb`.
+    /// - `b[48..96]` is $c_1$ serialized using `FqFormatLsb`.
+    ///
```

Comment:
Say: We often refer to this format as "least significant coefficient" (LSC) first.

< Then, add the polynomial example >

Reply:
Updated the documentation to mention it before the list.